openwisp_notifications/templates/openwisp_notifications/email_template.html

```diff
@@ -63,13 +117,23 @@
   <div class="background">
     <div class="container">
       <table style="width: 100%;">
-        <thead class="notification-header">
-        {% block notification_header %}
-          <tr>
+        <thead>
+          {% block notification_header %}
+          <!-- Large screen devices' notification header -->
```

Comment:
use django template comments instead of HTML comments, so we avoid sending comments in the email output

Reply:
I thought it will help in readability while debugging HTML.

Reply:
Yes it's a good idea. But Django templates have it's own commenting style which is ignored during rendering so it's good to use the style in Django templates. Commenting this way will still help in debugging

Reply:
:+1: